status-payloads-spec.md

```diff
@@ -173,16 +173,27 @@ The following messages types MUST be supported:
 
 #### Clock vs Timestamp and message ordering
 
+If a user sends a new message before the messages are delivered that where sent while the user was offline, the new
+message is supposed to be displayed last in a chat. This is where the basic algorithm of Lamport timestamp would fall
+short, as it's only meant to order causally related events.
+
+The status client therefore makes a "bid", speculating that it will beat the current chat-timestamp, s.t. the status client's
+Lamport timestamp format is: `clock = `max({timestamp}{post-id}, chat_clock + 1)`
```

Comment:
We only use `timestamp` now (in millisecond), `{post-id}` has been removed as not very useful.